src/OrderFormAcceptanceTests.TestData/Helpers/OrderItemHelper.cs

```diff
@@ -78,6 +78,26 @@ FROM dbo.CataloguePrice
             return result;
         }
 
+        public static async Task<ProvisioningType> GetProvisioningTypeForItemAsync(string itemId, string connectionString)
+        {
+            var query = @"SELECT ProvisioningTypeId
+                            FROM dbo.CataloguePrice
+                            WHERE CataloguePriceId = @itemId;";
+            var result = (await SqlExecutor.ExecuteAsync<ProvisioningType>(connectionString, query, new { itemId })).First();
+            return result;
+        }
+
+        public static async Task<string> GetCatalogueItemIdForNameAsync(string itemName, string bapiConnectionString)
+        {
+            var query = @"SELECT CatalogueItemId
+                        FROM dbo.CatalogueItem
+                        WHERE [Name] = @itemName;";
+
+            var result = (await SqlExecutor.ExecuteAsync<string>(bapiConnectionString, query, new { itemName })).First();
```

Comment:
Would `.Single` be better here too? Technically we could have multiple catalogue items with the same name but I suspect that could cause your tests to break? In which, perhaps `.Single` is better here on the fail-fast principle... Will leave up to you.